### PR TITLE
[release-4.2] Bug 1808505: only process install config changes to configmaps in the…

### PR DESCRIPTION
**Pull request by coreydaley**
… kubesystem namespace

### CODE REVIEW

**Comment by coreydaley** [comment]
/retest
infrastructure issues

**Comment by coreydaley** [comment]
@dmage @adambkaplan The other two prs have merged and this one is now ready to be reviewed.

File: pkg/operator/controller.go [hunk 1]
```diff
@@ -271,10 +272,15 @@ func (c *Controller) handler() cache.ResourceEventHandlerFuncs {
...
-			klog.V(1).Infof("add event to workqueue due to %s (add)", utilObjectInfo(o))
+			obj := o.(metav1.Object)
+			if obj.GetNamespace() == "kube-system" && obj.GetName() != "cluster-config-v1" {
+				return
+			}
+			klog.V(0).Infof("add event to workqueue due to %s (add)", utilObjectInfo(o))
 			c.workqueue.Add(workqueueKey)
 		},
 		UpdateFunc: func(o, n interface{}) {
+			//add event to workqueue due to *v1.ConfigMap, Namespace=kube-system, Name=cluster-config-v1
```
Comment (dmage) [inline pkg/operator/controller.go]: What is this comment for?

Comment (coreydaley) [reply pkg/operator/controller.go]: not sure where that came from :(
I will remove it

**Comment by coreydaley** [comment]
@dmage I removed the extraneous comment and set the loglevel back to 1 instead of 0 for the klog statements in the add/update/delete functions

**Comment by coreydaley** [comment]
/retest
Infra flake

**Comment by coreydaley** [comment]
@sdodson The associated bug priority has been updated.

**Pull request merged**